SDC/art.yaml

```diff
@@ -0,0 +1,29 @@
+config:
+  target: http://localhost:3000
+  phases:
+    -
+      duration: 90
+      arrivalRate: 1
+      name: phase 1
+    -
+      duration: 90
+      arrivalRate: 10
+      name: phase 10
+    -
+      duration: 90
+      arrivalRate: 100
+      name: phase 100
+    # -
+    #   duration: 90
+    #   arrivalRate: 1000
+    #   name: phase 1000
+  variables:
+    product_ids:
```

Comment:
You will probably want to randomize these. I believe that our stress test is supposed to hit the 'top' 10% of our database. I don't know exactly how many records you have but you could do something like `Math.floor(Math.random() * 1,000,000) + 9,000,000`